core/state_transition.go

```diff
@@ -289,10 +307,17 @@ func (st *StateTransition) TransitionDb() (*ExecutionResult, error) {
 	homestead := st.evm.ChainConfig().IsHomestead(st.evm.Context.BlockNumber)
 	istanbul := st.evm.ChainConfig().IsIstanbul(st.evm.Context.BlockNumber)
 	london := st.evm.ChainConfig().IsLondon(st.evm.Context.BlockNumber)
+	eip3860 := false
+	for _, eip := range st.evm.Config.ExtraEips {
+		if eip == 3860 {
+			eip3860 = true
+			break
+		}
+	}
```

Comment:
I don't think this iteration should happen in here. IIRC we already do this somewhere else?

Reply:
Yes in a private function in `vm` currently

Reply:
Made `HasEip3860()` a public method of `evm.Config`, this iteration is not repeated now.